code/modules/admin/admin_verbs.dm

```diff
@@ -950,7 +950,7 @@
 
 	var/datum/admin_help/AH = C.current_ticket
 
-	if(AH.tier == TICKET_ADMIN && !check_rights(R_ADMINTICKET, FALSE))
+	if(AH && AH.tier == TICKET_ADMIN && !check_rights(R_ADMINTICKET, FALSE))
```

Comment:
```suggestion
	if(!AH || (AH.tier == TICKET_ADMIN && !check_rights(R_ADMINTICKET, FALSE)))
```
and remove the check of AH below
Probably also add logging if a non-admin is trying to reply to an admin ticket.

Reply:
Or change it to deal with just normal messaging them if theres no ticket first as an early return

Reply:
That would break the logic. You want to private_message the person in the end no matter if there was an adminhelp or not.

Reply:
thats what i'm saying, if theres no adminhelp just call private_message in an early return then deal with the case where it is an admin help with more simplified logic

Reply:
The message is not yet defined at this point, changing the logic here would also break the marking behavior. Is it the most optimized code? No, but this allows for a lot of admin QoL and 3 more simple checks won't hurt.

Reply:
Ah okay I see now, yeah would require more work